metadata/metadata_service/proxy/neo4j_proxy.py

```diff
@@ -1785,7 +1771,7 @@ def _exec_feature_query(self, *, feature_key: str) -> Dict:
             'owner_tags': owner_tags,
             'tags': tags,
             'badges': self._make_badges(feature_records.get('badge_records')),
-            'partition_column': partition_column,
+            'partition_column': None,
```

Comment:
can we just remove this instead? It will be in prog descriptions later, if we add it.